MAUI/ImageEditor/save.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: Image save in .NET MAUI ImageEditor control | Syncfusion
+description: Learn about save support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Save in .NET MAUI ImageEditor (SfImageEditor)
```

Comment:
# Save image using .NET MAUI Image Editor (SfImageEditor)

Reply:
Changed the heading